docs/bucket/replication/README.md

```diff
@@ -154,6 +155,13 @@ The status of replication can be monitored by configuring event notifications on
 On the target bucket, `s3:PutObject` event shows `X-Amz-Replication-Status` status of `REPLICA` in the metadata. Additional metrics to monitor backlog state for the purpose of bandwidth management and resource allocation are  
 an upcoming feature.
 
+### Sync/Async Replication
+By default, replication is completed asynchronously. If synchronous replication is desired, set the --sync flag while adding a
+remote replication target using the `mc admin bucket remote add` command
+```
+ mc admin bucket remote add myminio/srcbucket https://accessKey:secretKey@replica-endpoint:9000/destbucket --service replication --region us-east-1 --sync
+```
```

Comment:
```suggestion
 mc admin bucket remote add myminio/srcbucket https://accessKey:secretKey@replica-endpoint:9000/destbucket --service replication --region us-east-1 --sync --healthcheck-seconds 100
```
```

Also the healthcheck value cannot be below 1sec

Reply:
done, set the default if unspecified to 60 secs.